examples/splunk-sink/README.md

```diff
@@ -0,0 +1,62 @@
+# Splunk sink example
+
+The solution helps you set up a log-streaming pipeline from Stackdriver Logging to Splunk.
+
+The example is for a project-level sink, but it can be easily be adapted for aggregated log sinks.
+
+## Instructions
```

Comment:
Do we have a quick way to stand up a Splunk instance within GCE to verify this? Can we automate this part of the process in the example or is this entirely manual?

Reply:
Typically users are running Splunk on-prem. Installing Splunk itself is out-of-scope and I don't think we need to automate this process.

Reply:
Installing Splunk in this example is definitely overkill. In terms of review I'd like to check out the setup process for correctness, but if it's a significant amount of work (and I suspect that it will be) we can skip that part of the review.

Reply:
Let's skip. I'm confident that part of the process is correct since it mirrors our existing docs.